docs/tables/datadog_dashboard.md

```diff
@@ -0,0 +1,20 @@
+# Table: datadog_dashboard
+
+A dashboard is Datadog’s tool for visually tracking, analyzing, and displaying key performance metrics.
+
+## Examples
```

Comment:
Can you please add more 2-3 more examples around listing the roles that can edit the dashboard and listing useful widget information?